pages/profile.tsx

```diff
@@ -15,25 +15,26 @@ export default function Profile() {
   } else {
     // map used rows to appropriate squares
     board = (
-      <div>
+      <Container>
```

Comment:
I personally found that the profile page was already formatted fine on mobile for me -- the page that gave me issues was the statistics screen, which still has those issues